src/css/style.css

```diff
@@ -0,0 +1,197 @@
+
+body{
+  background: #DBC2F5;
+
+}
+/* Navbar styles */
+.navbar {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  background-color: transparent;
+  padding: 20px;
+  width: 100%;
+  max-width: 1200px;
+  margin: 0 auto;
+}
+
+/* Logo styles */
+.logo img {
+  width: 100px;
+  height: auto;
+}
+
+/* Navigation container styles */
+.navcontainer {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  flex-grow: 1;
+}
+
+/* Navigation list styles */
+.navcontainer ul {
+  display: flex;
+  list-style-type: none;
+  margin: 0;
+  padding: 0;
+}
+
+.navcontainer li {
+  font-size: 16px;
+  color: black;
+  font-family: DM Sans, Regular;
+  margin-right: 20px;
+}
+
+.menu{
+  display: none;
+}
+
+
+
+
+
+
+
+
+
+
+
+
+
```

Comment:
remove extra spaces

Reply:
Thank you for the feedback. i have made the changes required